src/containers/System/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Tabs } from 'antd';
+import Container from 'components/Container';
+import styles from './index.module.scss';
+
+const System = ({ children }) => {
+  const location = useLocation();
+
+  const { TabPane } = Tabs;
+
+  return (
+    <div>
+      <div className={styles.System}>
+        {location.pathname === '/system/manufacturer' ||
+        location.pathname === '/system/firmware' ? (
```

Comment:
change expression to ` {(location.pathname === '/system/manufacturer' ||
        location.pathname === '/system/firmware') && (`